BookingManager-api/src/main/java/cz/muni/fi/pa165/brown/dto/HotelDTO.java

```diff
@@ -0,0 +1,97 @@
+package cz.muni.fi.pa165.brown.dto;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ *
+ * @author Michal Hagara
+ */
+
+public class HotelDTO
+{
+
+    private Long id;
+
+    @NotNull
```

Comment:
I think that these NotNull annotations are not needed since this is already managed on dao layer.